integration-tests/constants/requestData.ts

```diff
@@ -17,9 +17,25 @@ import {
   PII_FORM_REQUEST_STATUS_FIELD_ID
 } from './zendeskParameters'
 
+const generateSubjectLine = () => {
+  const fixedSubjectLine = process.env.FIXED_SUBJECT_LINE
+  if (fixedSubjectLine) {
+    return fixedSubjectLine
+  }
+  return `Integration Test Request - ` + generateRandomNumber()
```

Comment:
Although fine could this be re-written as: 
``
  `Integration Test Request - ${generateRandomNumber()}`
``
Just seems weird that as a template string it's not all together.

Reply:
Yes, it came from another part of the codebase. I'll change it, it's silly.